<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_project">
     <property name="title">
      <string>Open project</string>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/icons/Ressources/open_file.png</normaloff>:/icons/Ressources/open_file.png</iconset>
     </property>
     <addaction name="actionBrowse"/>
    </widget>
    <addaction name="actionNew_project"/>
    <addaction name="menuOpen_project"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_all"/>
    <addaction name="actionExport_standalone"/>
    <addaction name="actionClose_project"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionPlay"/>
   </widget>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>Management</string>
    </property>
    <addaction name="actionDatas_manager"/>
    <addaction name="actionSystems_manager"/>
    <addaction name="actionVariables_manager"/>
    <addaction name="actionPictures_manager"/>
    <addaction name="actionKeyboard_controls"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSet_BR_path_folder"/>
   </widget>
   <widget class="QMenu" name="menuSpecials">
    <property name="title">
     <string>Special elements</string>
    </property>
    <addaction name="actionAutotiles"/>
    <addaction name="actionSprite_walls"/>
    <addaction name="action3D_objects"/>
    <addaction name="actionReliefs"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionShow_Hide_grid"/>
    <addaction name="actionShow_Hide_square_informations"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdition"/>
   <addaction name="menuManagement"/>
   <addaction name="menuSpecials"/>
   <addaction name="menuOptions"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_project"/>
   <addaction name="actionBrowse"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_all"/>
   <addaction name="separator"/>
   <addaction name="actionDatas_manager"/>
   <addaction name="actionSystems_manager"/>
   <addaction name="actionVariables_manager"/>
   <addaction name="actionPictures_manager"/>
   <addaction name="actionKeyboard_controls"/>
   <addaction name="separator"/>
   <addaction name="actionAutotiles"/>
   <addaction name="actionSprite_walls"/>
   <addaction name="action3D_objects"/>
   <addaction name="actionReliefs"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the engine</string>
   </property>
   <property name="statusTip">
    <string>Quit the engine</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew_project">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/new_file.png</normaloff>:/icons/Ressources/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New project...</string>
   </property>
   <property name="iconText">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/open_file.png</normaloff>:/icons/Ressources/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open an existing project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_project">
   <property name="text">
    <string>Close project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/save.png</normaloff>:/icons/Ressources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/play.png</normaloff>:/icons/Ressources/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play your game!</string>
   </property>
  </action>
  <action name="actionDatas_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/datas.png</normaloff>:/icons/Ressources/datas.png</iconset>
   </property>
   <property name="text">
    <string>Datas manager</string>
   </property>
   <property name="iconText">
    <string>Datas</string>
   </property>
   <property name="toolTip">
    <string>Datas manager of your project</string>
   </property>
  </action>
  <action name="actionVariables_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/datas.png</normaloff>:/icons/Ressources/datas.png</iconset>
   </property>
   <property name="text">
    <string>Variables manager</string>
   </property>
   <property name="iconText">
    <string>Variables</string>
   </property>
  </action>
  <action name="actionSystems_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/datas.png</normaloff>:/icons/Ressources/datas.png</iconset>
   </property>
   <property name="text">
    <string>Systems manager</string>
   </property>
   <property name="iconText">
    <string>Systems</string>
   </property>
  </action>
  <action name="actionKeyboard_controls">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/inputs.png</normaloff>:/icons/Ressources/inputs.png</iconset>
   </property>
   <property name="text">
    <string>Keyboard controls</string>
   </property>
   <property name="iconText">
    <string>Keyboard</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/save.png</normaloff>:/icons/Ressources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save all</string>
   </property>
   <property name="iconText">
    <string>All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExport_standalone">
   <property name="text">
    <string>Export standalone...</string>
   </property>
  </action>
  <action name="actionPictures_manager">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/datas.png</normaloff>:/icons/Ressources/datas.png</iconset>
   </property>
   <property name="text">
    <string>Pictures manager</string>
   </property>
   <property name="iconText">
    <string>Pictures</string>
   </property>
  </action>
  <action name="actionSet_BR_path_folder">
   <property name="text">
    <string>Set BR path folder...</string>
   </property>
  </action>
  <action name="actionAutotiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/datas.png</normaloff>:/icons/Ressources/datas.png</iconset>
   </property>
   <property name="text">
    <string>Autotiles</string>
   </property>
  </action>
  <action name="actionSprite_walls">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/datas.png</normaloff>:/icons/Ressources/datas.png</iconset>
   </property>
   <property name="text">
    <string>Sprite walls</string>
   </property>
   <property name="iconText">
    <string>Walls</string>
   </property>
  </action>
  <action name="action3D_objects">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/datas.png</normaloff>:/icons/Ressources/datas.png</iconset>
   </property>
   <property name="text">
    <string>3D objects</string>
   </property>
   <property name="iconText">
    <string>3D objects</string>
   </property>
  </action>
  <action name="actionReliefs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/datas.png</normaloff>:/icons/Ressources/datas.png</iconset>
   </property>
   <property name="text">
    <string>Reliefs</string>
   </property>
  </action>
  <action name="actionShow_Hide_grid">
   <property name="text">
    <string>Show / Hide grid</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionShow_Hide_square_informations">
   <property name="text">
    <string>Show / Hide square informations</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
